src/main/java/seedu/bankwithus/Parser.java

```diff
@@ -24,18 +31,30 @@ public void parseUserInput(String input) throws CommandNotFoundException {
         String args = split.length == 2 ? split[1] : "";
         Ui screen = new Ui();
         switch (command) {
-        case "exit":
-            bwu.isExitEntered = true;
-            break;
-        case "view-account":
-            String accDetails = bwu.accounts.getAllAccountDetails();
-            screen.viewAccount(accDetails);
-            break;
-        default:
-            throw new CommandNotFoundException();
+            case "exit":
+                bwu.isExitEntered = true;
+                break;
+            case "view-account":
+                String accDetails = bwu.accounts.getAllAccountDetails();
+                screen.viewAccount(accDetails);
+                break;
+            case "withdraw":
+                float final_bal = parseWithdrawAmt(args);
+                if(final_bal > -1) {
```

Comment:
Wait why not >= 0? Currently, since final_bal is float, >-1 can mean -0.2, which is still invalid